src/locals/hu.json

```diff
@@ -0,0 +1,145 @@
+            "line1": "Végül írunk egy",
+            "emphasis1": "Helyettesítő gondolatot."
+        },
+        "show_off": {
+            "line1": "Találd meg később is:"
+        },
+        "got_it": {
+            "line1": "Tényleg csak ennyi",
+            "btn_title": "Uccu neki!"
+        }
+    },
+    "auto_thought": "Automatikus gondolat",
+    "cog_distortion": "Kognitív Torzítás",
+    "challenge": "Challenge",
+    "alt_thought": "Helyettesítő gondolat",
+    "cbt_form": {
+        "edit": "Szerkeszt",
+        "new": "Új",
+        "save": "Ment",
+        "auto_thought_placeholder": "Mi történik most a fejedben?",
+        "cog_distortion": "Kognitív torzítás",
+        "challenged": "Vitat",
+        "changed_placeholder": "Vitasd meg a gondolatot!",
+        "alt_thought_placeholder": "Mire gondolhatnánk helyette?"
+    },
+    "all_or_nothing_thinking": "Mindent vagy semmit gondolkogás.",
+    "all_or_nothing_thinking_explanation": "Ez a torzítás akkor történik, amikor nem tudunk árnyalatokban gondolkodni. Ha egy kis hibát fedezünk fel magunkban és emiatt az gondoljuk velejünkig romlottak vagyunk, akkor a végletekben való gondolkozás torzítását éljük át.",
+    "all_or_nothing_thinking_thought": "Elbaltáztam az interjúmat, sehova se fognak felvenni.",
+    "catastrophizing": "Katasztrófizálunk",
+    "catastrophizing_explanation": "Ha a bolhából is elefántots csinálunk, akkor katasztofizálunk. Ha egy kis hibát ejtettél a munkahelyeden, még nem biztos, hogy csak a miatt kirúgnak.",
+    "catastrophizing_thought": "Ideges vagyok, biztos szívrohamom van.",
+    "emotional_reasoning": "Érzelmi érvelés",
+    "emotional_reasoning_explanation_1": "\"Érzem, tehát biztos igaz is.\" Ha jogosnak érzed a \"veszélyt\" azért, mert te félsz tőle, akkor elképzelhető, hogy érzelmi érvelést alkalmazol. A dolgok nem azért veszélyesek, mert mi félünk tőle vagy azt gondoljuk, hogy rossz.",
+    "emotional_reasoning_explanation_2": "Ezt sokszor nehéz érzékelni. Nehéz összeszedned magad annyira, hogy logikusan tudj gondolkozni az érzelmeidről is.",
+    "emotional_reasoning_thought": "Bűnösnek érzem magam, ezért biztos, hogy rosz fát tettem a tűzre.",
+    "fortune_telling": "Jövendőmondás",
+    "fortune_telling_explanation": "Amikor ki akarjuk találni a jövőt. Sokszor túlbecsüljük a jövendőmondó képességünket. Sokszor erős előfeltételezésekkel élünk és csak az azt alátámasztó bizonyítékokat vesszük figyelembe.",
+    "fortune_telling_thought": "Le fog zuhanni a repülő, amin ülök.",
+    "labeling": "Címkézés",
+    "labeling_explanation": "Ha egy jellegzetes tulajdonsággal azonosítunk egy embert, akkor címkézünk. Ha valaki bevágott eléng, attól még nem biztos, hogy ő \"*eggfej,\" lehet, hogy csak siet. Ez ránk is vonatkozik; csak azért, mert hibázunk még nem vagyunk \"hibásak.\"",    
+    "labeling_thought": "Elrontottam a ZH-t/TZ-t, biztos rossz tanuló vagyok.",
+    "magnification_of_the_negative": "A Negatívumok felerősítése",
```

Comment:
for consistency: maybe capitalize both of these? or just go with "Negatívumok felerősítése", so that it's just the start of the sentence? (English seems to like Capitalized Titles a lot more.)